cruise-control-client/cruisecontrolclient/client/CCParameter/BooleanParameter.py

```diff
@@ -2,6 +2,7 @@
 
 from cruisecontrolclient.client.CCParameter.Parameter import AbstractParameter
 
+set_of_choices = {'true', 'false'}
```

Comment:
Would it be possible to make set of choices `case-insensitive` (unless they already are)?

Reply:
(whoops, comment then resolve, not the other way around 😛)

Why [yes](https://stackoverflow.com/a/27616814/11280049), it is possible to have `argparse` lowercase the provided values for these switches, so that they will be compared against (and provided to `cruise-control` as) lowercase strings.

db11aacf9f2c826980086e47de59362f602d3429 should provide this functionality.